apps/opik-backend/src/main/java/com/comet/opik/domain/TraceDAO.java

```diff
@@ -238,17 +239,77 @@ INSERT INTO traces (
 
     private static final String SELECT_BY_ID = """
             SELECT
-                *
-            FROM
-                traces
-            WHERE id = :id
-            AND workspace_id = :workspace_id
-            ORDER BY last_updated_at DESC
-            LIMIT 1
+                t.id,
+                t.workspace_id,
+                t.project_id,
+                t.name,
+                t.start_time,
+                t.end_time,
+                t.input,
+                t.output,
+                t.metadata,
+                t.tags,
+                t.created_at,
+                t.last_updated_at,
+                t.created_by,
+                t.last_updated_by,
```

Comment:
Normally, I don't like `*`, but in this case, it would prevent the query from breaking once we add new fields in the traces table.

```suggestion
                t.*,
```

Reply:
Sure, this is more concise and maintainable. Done in next revision.

There are a couple of similar queries in other models, we'll do the same in the future.